docs/src/pages/[platform]/getting-started/usage/nextjs/react.mdx

```diff
@@ -24,6 +24,234 @@ Then, install the Amplify UI React package:
+      import { Button } from './AmplifyUIComponents';
+
+      export default function Home() {
+        return <Button>Hello, Amplify!</Button>;
+      }
+      ```
+
+      </ExampleCode>
+    </Example>
+
+To use more Amplify UI components, you need to re-export them in the same client module `AmplifyUIComponents.jsx` and then load them from there.
+
+    <ExampleCode>
+
+    ```tsx
+    // AmplifyUIComponents.jsx
+    'use client';
+
+    import { Alert, Button, Card } from '@aws-amplify/ui-react';
+    import '@aws-amplify/ui-react/styles.css';
+
+    export { Alert, Button, Card };
+    ```
+
+    </ExampleCode>
+#### Connected Components
+    To use Amplify UI Connected Components in app directory, you must set up `Amplify.configure` in a client module.
+
+    First, create a client module, for example, `configure.js` by placing `'use client'` at the top of the file under `app` directory:
+
+    <ExampleCode>
+
+    ```tsx
+    // configure.js
+    "use client";
+
+    import { Amplify } from "aws-amplify";
+
+    import config from "../src/aws-exports";
+    
+    Amplify.configure(config);
+    ```
+
+    </ExampleCode>
+
+    Then import `configure.js` in `page.jsx` to finish the setup,
+        <ExampleCode>
+
+    ```tsx
+    // page.jsx
+    import "./configure";
+
+    export default function Home() {
+      return <h1>Hello, Amplify!</h1>;
+    }
+    ```
+
+    </ExampleCode>
+
+Like UI components, you must re-export Connected Components as Client Components before using them.
+
+An example below is shown for `Authenticator` component:
+
+First create a client module, for example, `AmplifyConnectedComponents.jsx` by placing `'use client'` at the top of the file under `app` directory,
+
+    <ExampleCode>
+
+    ```tsx
+    // AmplifyConnectedComponents.jsx
+    'use client';
+
+    import { Authenticator } from '@aws-amplify/ui-react';
+    import '@aws-amplify/ui-react/styles.css';
+   
+    export { Authenticator };
+    ```
+
+    </ExampleCode>
+
+Then, import `Authenticator` from the client module and use it anywhere you want, e.g., `app/page.jsx`.
+
+    <Example>
+      <ExampleCode>
+
+      ```tsx
+      // app/page.jsx
+      import { Authenticator } from './Authenticator';
+
+      export default function Home() {
+        return <Authenticator>Hello, Amplify!</Authenticator>;
+      }
+      ```
+
+      </ExampleCode>
+    </Example>
+
+To use more Connected components, you need to re-export them in the same client module `AmplifyConnectedComponents.jsx` and then load them from there.
+
+    <ExampleCode>
+
+    ```tsx
+    // AmplifyConnectedComponents.jsx
+    'use client';
+
+    import { Authenticator } from '@aws-amplify/ui-react';
+    import { StorageManager } from '@aws-amplify/ui-react-storage';
+    import '@aws-amplify/ui-react/styles.css';
+
+    export { Authenticator, StorageManager };
+    ```
+
+    </ExampleCode>
+
+**You may want to combine the setup and re-exports into a single client module.**
```

Comment:
TBH, I kinda like this way better. It seems weird to have a separate file to do the `Amplify.configure` and then not export anything? Should we just show this way? I don't think we need to show 2 options, its a bit verbose. I can also be wrong about this if you disagree.

Reply:
You are totally right and this my original opinion about how to configure. We went back and forth with this part and we got feedback from JS team that in cases where users would use SSR features, they need the config in a Server Component as well. So we might as well show a pattern to let users know, the config can be separated.